packages/connector/jest.setup.js

```diff
@@ -1,14 +1,11 @@
 // eslint-disable-next-line @typescript-eslint/no-var-requires
-const IORedis = require('ioredis')
-
-const REDIS_URL = process.env.REDIS || 'redis://127.0.0.1:6380'
-const redis = new IORedis(REDIS_URL, { lazyConnect: true })
+const { GenericContainer } = require('testcontainers')
 
 module.exports = async () => {
-  await redis.connect()
-  if (redis.status === 'ready') {
-    await redis.disconnect()
-  } else {
-    throw new Error('expected redis at ' + REDIS_URL)
-  }
+  const redisContainer = await new GenericContainer('redis')
+    .withExposedPorts(6379)
+    .start()
+
+  global.__CONNECTOR_REDIS__ = redisContainer
+  process.env.REDIS = `redis://localhost:${redisContainer.getMappedPort(6379)}`
```

Comment:
The other files use constant variables for the port number.  Could you do the same here to be consistent?